.gitignore

```diff
@@ -2,4 +2,5 @@ __pycache__/
 Sandbox/
 jes-6.0-linux*
 *.pyc
+Tests/test_Scale.py
```

Comment:
If this is testing functions in the Picture class, then can we merge this file into the test_Picture.py file?